core/src/test/java/org/apache/metamodel/QueryPostprocessDataContextTest.java

```diff
@@ -1022,4 +1022,16 @@ public void testQueryWithDotInTableName() throws Exception {
 
         assertEquals("file.csv.foo = 'bar'", item.toSql());
     }
+    
+    public void testQueryWithMultipleColumnsInExpression() {
+        Query query1 = new Query().from(table1).select("contributor_id,name");
+        DataSet set = getDataContext().executeQuery(query1);
+        assertEquals(set.next(), true);
+        assertEquals(set.getRow().toString(), "Row[values=[1, kasper]]");
```

Comment:
Not that it matters much, but I am pretty sure that "expected" and "actual" parameters are switched around here. Normally you have expectation as the first argument.